templates/zerver/footer.html

```diff
@@ -1,186 +1,111 @@
-                </a>
-            </li>
-            <li>
+                <a href="/development-community/" target="_blank">{{ _("Community chat") }}</a>
                 <a href="/help/contact-support">{{ _("Contact support") }}</a>
-            </li>
-            <li>
-                <a href="https://blog.zulip.com/">{{ _("Blog") }}</a>
-            </li>
-        </ul>
-    </div>
-    <div class="footer-section">
-        <h3>{{ _("For administrators") }}</h3>
-        <ul>
-            <li>
+            </p>
+            <p>
                 <a href="/help/getting-your-organization-started-with-zulip">
-                    {{ _("Setting up your organization") }}
+                {{ _("Organization set up") }}
                 </a>
-            </li>
-            <li>
                 <a href="https://zulip.readthedocs.io/en/stable/production/install.html">
-                    {{ _("Installing a Zulip server") }}
+                {{ _("Installing a Zulip server") }}
                 </a>
-            </li>
-            <li>
                 <a href="https://zulip.readthedocs.io/en/stable/production/upgrade-or-modify.html">
-                    {{ _("Upgrading a Zulip server") }}
+                {{ _("Upgrading a Zulip server") }}
                 </a>
-            </li>
-            <li>
                 <a href="https://zulip.readthedocs.io/en/latest/production/upgrade-or-modify.html#modifying-zulip">
-                    {{ _("Modifying Zulip") }}
+                {{ _("Modifying Zulip") }}
                 </a>
-            </li>
-        </ul>
-    </div>
-    <div class="footer-section">
-        <h3>{{ _("For contributors") }}</h3>
-        <ul>
-            <li><a href="https://github.com/zulip/zulip/">{{ _("GitHub") }}</a></li>
-            <li>
+            </p>
+        </div>
+        <div class="footer__section">
+            <span class="footer__section-title">
+                {{ _("CONTRIBUTING") }}
+            </span>
+            <p>
                 <a href="https://zulip.readthedocs.io/en/latest/contributing/contributing.html">
-                    {{ _("Contributing guide") }}
+                {{ _("Contributing guide") }}
                 </a>
-            </li>
-            <li><a href="/development-community/">{{ _("Development community") }}</a></li>
-            <li>
+                <a href="/development-community/">{{ _("Development community") }}</a>
                 <a href="https://zulip.readthedocs.io/en/latest/translating/translating.html">
-                    {{ _("Translation") }}
+                {{ _("Translation") }}
                 </a>
-            </li>
-            <li><a href="/api/">REST API</a></li>
-        </ul>
+                <a href="/api/">API</a>
+                <a href="https://github.com/zulip/zulip/">{{ _("GitHub") }}</a>
+            </p>
+        </div>
+        <div class="footer__section">
+            <span class="footer__section-title">
+                {{ _("ABOUT US") }}
```

Comment:
Do we want these tagged for translation in all-caps form, or to use CSS to capitalize the section titles?

Reply:
If it makes a difference I can change it but I don't think it should.